addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -121,8 +118,6 @@
 "","","","","","","","","","account_tax_template_s_iva21b","account_tax_template_s_req52","",""
 "","","","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_iva21s","",""
 "","","","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_req52","",""
-"","","","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_iva21isp","",""
```

Comment:
So we delete a tax in the CoA.  I think it will only impact the new CoAs, but code could depend on the xml_id, so it is not ideal.  On the other hand, if the tax is wrong, it is wrong.

Reply:
The "21% RC" Sales tax was doing nothing different from the "21% G" Sales tax, so I just deleted it.
And the fiscal position mapping is just the same, so it's useless. Also saying it's a non-0% but saying it's a sale Reverse Charge makes no sense ?